docs/UserGuide.md

```diff
@@ -454,7 +454,9 @@ Format: **`import`** `FILE_NAME.json`
 * File must be located in the `./data` directory in relative to the directory of `Socius.jar`.
 
 Examples:
-* **`import`** `amy.json` import a contact book via a file named `amy.json` into your existing application.
+* **`import`** `import.json` imports a contact book via a file named `import.json` into your existing application.
```

Comment:
Actually, I have been thinking "import import.json" might not be very intuitive for first-time users, cos there is 2 'import's. Maybe we can have import groupmates.json, so it's more intuitive.

Reply:
yeah that's a better idea actually